styx-common/src/test/java/com/spotify/styx/storage/DatastoreTransactionalStorageTest.java

```diff
@@ -0,0 +1,101 @@
+/*-
+ * -\-\-
+ * Spotify Styx Common
+ * --
+ * Copyright (C) 2016 - 2018 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.styx.storage;
+
+import static org.junit.Assert.assertFalse;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Key;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.cloud.datastore.Transaction;
+import com.google.cloud.datastore.testing.LocalDatastoreHelper;
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DatastoreTransactionalStorageTest {
+
+  private static LocalDatastoreHelper helper;
+  private static Datastore datastore;
+
+  @BeforeClass
+  public static void setUpClass() throws Exception {
+    helper = LocalDatastoreHelper.create(1.0); // 100% global consistency
```

Comment:
Any reason to go to max consistency? I think in testing we might want something closer to the opposite, max chaos?

Reply:
Perhaps 100% consistency should be the default testing behaviour and that value decreased for specific tests around strong/weak consistency.

Reply:
Hm, my point was indeed about the default testing behavior, by default we should try to catch things we didn't think of. A piece of code that we _thougt_ wasn't affected by weak consistency, might actually _be_ affected.